recipes/glad/2.x/conanfile.py

```diff
@@ -0,0 +1,147 @@
+import os
+import sys
+from pathlib import Path
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import collect_libs, copy, get
+from conan.errors import ConanInvalidConfiguration
+
+class GladConan(ConanFile):
+    name = "glad"
+    description = "Multi-Language GL/GLES/EGL/GLX/WGL Loader-Generator based on the official specs."
+    topics = ("opengl",)
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Dav1dde/glad"
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "no_loader": [True, False],
+        "spec": ["gl", "egl", "glx", "wgl"], # Not relevant for 2.x. A spec will be included unless its version is None
+        "extensions": ["ANY"], # A comma separated list of extensions, if missing all extensions are included
+        "debug": [True, False], # Enable the additional GLAD Debugging layer
+        "multicontext": [True, False],
+        "gl_profile": ["compatibility", "core"],
+        "gl_version": ["None", "1.0", "1.1", "1.2", "1.3", "1.4", "1.5", "2.0",
+                       "2.1", "3.0", "3.1", "3.2", "3.3", "4.0", "4.1", "4.2",
+                       "4.3", "4.4", "4.5", "4.6"],
+        "gles1_version": ["None", "1.0"],
+        "gles2_version": ["None", "2.0", "3.0", "3.1", "3.2"],
+        "glsc2_version": ["None", "2.0"],
+        # if specification is egl
+        "egl_version": ["None", "1.0", "1.1", "1.2", "1.3", "1.4", "1.5"],
+        # if specification is glx
+        "glx_version": ["None", "1.0", "1.1", "1.2", "1.3", "1.4"],
+        # if specification is wgl
+        "wgl_version": ["None", "1.0"]
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "no_loader": False,
+        "spec": "gl",
+        "extensions": "",
+        "debug": False,
+        "multicontext": False,
+        "gl_profile": "compatibility",
+        "gl_version": "3.3",
+        "gles1_version": "None",
+        "gles2_version": "None",
+        "glsc2_version": "None",
+        "egl_version": "None",
+        "glx_version": "None",
+        "wgl_version": "None"
```

Comment:
```suggestion
        "gles1_version": None,
        "gles2_version": None,
        "glsc2_version": None,
        "egl_version": None,
        "glx_version": None,
        "wgl_version": None
```

This was a limitation from conan1